### ISSUE TITLE
Delete publication from orcid profile

### ISSUE THREAD

**Issue by kaplun** [inveniosoftware/invenio-orcid#14]
Say author X claims publication P and this information is reflected into her ORCID profile.
In a second moment it is realized the above was a mistake, and the claim is cancelled. 

We need that the publication P is then automatically deleted from X's ORCID profile.

**Comment by kaplun** [comment]
@Panos512 I don't think this use case is already covered: papers are deleted from ORCID only when they are fully deleted from the original invenio instance, right?

**Comment by Panos512** [comment]
Yes at this moment `delete_from_orcid` runs on `before_record_delete` signal. But we can plug it to either the claiming code itself or after a `claiming signal` is sent.

**Comment by kaplun** [comment]
Mmh. Actually today, upon record update I guess you are pushing the record to all those users for which the paper has been claimed. So I guess we can query the the model and delete the very record for all those users for which we have sent it in the past and we are no longer sending it.

**Comment by Panos512** [comment]
Yes indeed we will need to `refresh` the record for every user we pushed on his behalf. I'll go on and implement this.
